code/modules/mob/living/silicon/robot/drone/drone_abilities.dm

```diff
@@ -49,6 +49,21 @@
 	else
 		..()
 
+/mob/living/silicon/robot/drone/verb/self_repair()
+	set name = "Self Repair"
+	set category = "Drone"
+	set desc = "Attempts to repair damage to our body. You will have to remain motionless until repairs are complete."
```

Comment:
These two sentences are gramatically awkward as you use `our` in one, and `You` in the other.